docs/smart-contracts/creator-tools/ProtocolSDK.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Protocol SDK
+---
+
+##### Get started building with the Zora smart contracts
+---
+
+The Protocol SDK makes interacting with the ZORA protocol easier by generating required transactions for our contracts.
```

Comment:
few things missing from here:
* its a js/typescript sdk
* it works with viem/wagmi.
* it interacts with our backend as well as smart contracts